test/Chainweb/Test/RestAPI/Utils.hs

```diff
@@ -267,6 +282,10 @@ polling sid cenv rks pollingExpectation =
       Just cr ->  _crReqKey cr == rk && validate (_crResult cr)
       Nothing -> False
 
+getCurrentBlockHeight :: ChainwebVersion -> ClientEnv -> ChainId -> IO (Maybe BlockHeight)
+getCurrentBlockHeight сv cenv cid = do
+  cuts <- either (const $ error "failed to get cuts") id <$> runClientM (cutGetClient сv) cenv
+  return $ _bhwhHeight <$> HM.lookup cid (_cutHashes cuts)
```

Comment:
I think it's fine here to return `IO BlockHeight`, I don't see why we'd call this function with a `cid` that doesn't exist on the `cv` at the time.